### PR TITLE
New package: python3-aiofiles-0.5.0, gns3-server: fix dependencies

### PR DESCRIPTION
**Pull request by krkk**

### CODE REVIEW

File: srcpkgs/gns3-server/template [hunk 1]
```diff
@@ -18,7 +17,9 @@ checksum=51a7b5b9701a29d6ef1003d9c90a4d098e1d7fa03df3c66b74e70a729f9f8b59
 pre_build() {
 	# relax requirements
 	sed -i '/typing/d' requirements.txt
+	sed -i 's|aiofiles.*|aiofiles>=0.4.0|' requirements.txt
```
Comment (abenson) [inline srcpkgs/gns3-server/template:20]: consider using `vsed`

**Comment by Chocimier** [comment]
Hello, are you still interested in this?

New version of gns3-server got released by the time, could you update?
Could you also propose authors to relax required versions?

**Comment by krkk** [comment]
> Hello, are you still interested in this?

I don't use it anymore, but I can look at this.

> New version of gns3-server got released by the time, could you update?

Yeah, sure.

> Could you also propose authors to relax required versions?

well, sigh…

**Comment by Chocimier** [comment]
Thanks! Some changes added, will merge if everything is ok.

**Comment by krkk** [comment]
Looks good to me, thanks

**Pull request merged**